lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -171,7 +171,7 @@ def open_jdbc_connection
       @database.test_connection
     rescue Sequel::DatabaseConnectionError => e
       @logger.warn("Failed test_connection.")
-      @database.close_jdbc_connection
```

Comment:
hello, after doing all the changes you have claimed to be done, i get another error, which says Bundler::GemNotFound: Could not find logstash-input-jdbc-4.2.1 in any of the sources 

I tried to bundle install  but it doesnt recognise it as an internal comand. Could you help me ?